src/ExtendedTableWidget.cpp

```diff
@@ -64,6 +64,7 @@ ExtendedTableWidget::ExtendedTableWidget(QWidget* parent) :
     QTableView(parent)
 {
     setHorizontalScrollMode(ExtendedTableWidget::ScrollPerPixel);
+    setVerticalScrollMode(ExtendedTableWidget::ScrollPerItem);
```

Comment:
The problem was caused by the default vertical scroll mode, which is set to
`ScrollPerPixel` on macOS, and `ScrollPerItem` on other systems.